lambdas/instance-types/template.yaml

```diff
@@ -0,0 +1,136 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
too much stuff in this file.  please follow pattern in  https://github.com/Sage-Bionetworks/aws-infra/blob/master/lambdas/cfn-explode-macro/template.yaml